docs/source/cluster_management/logging.rst

```diff
@@ -0,0 +1,14 @@
+.. _logging-configuration:
```

Comment:
Though this is more general for all KVS types, it might make sense to combine this with the Cassandra KVS tracing docs in https://github.com/palantir/atlasdb/blob/develop/docs/source/configuration/enabling_cassandra_tracing.rst